### DISCUSSION TITLE
Getting bugs using the given tensor example in ReadMe

### DISCUSSION THREAD

**Discussion by harjot20022001** [boostorg/ublas#148] [Q&A]
I tried to use the example given in https://github.com/boostorg/ublas#simple-example and I'm getting bugs on my system. The keywords 'index','tenson_dynamics' and 'ones' are unrecognized. Is it a bug or anything else?

**Comment by bassoy** [comment]
Thanks for using uBlas and sorry for the late answer. Few questions:
- How did you download uBlas?
- Which branch are you using?

**Reply by harjot20022001** [reply]
I cloned the develop branch and I'm using MSVS. I did include the correct directories still can't resolve my issue. Sorry for the late reply as I had given up on it.

**Reply by amitsingh19975** [reply]
Can you share a screenshot of the `include directories` dialogue box? Are you including the root directory that you cloned or something else? If MSVC doesn't work, you can use the command line to compile using `cl`.

```bat
cl /std:c++20 /I PATH_TO_CLONE_UBLAS\\include CPP_SOURCE_FILE
```
Something like this.
